### PR TITLE
Add dbxcli 3.0.0

### PR DESCRIPTION
**Pull request by Slach**
dbxcli - A command line client for Dropbox

### CODE REVIEW

**Comment by Slach** [comment]
@jasongodev how to add `Autoupdate Hash Extraction`?

dbxcli doesn't have hash file in releases artifact

**Comment by Slach** [comment]
@jasongodev any news?

File: bucket/dbxcli.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        }
+    },
+    "bin": [
+        [
+            "dbxcli-windows-amd64.exe",
+            "dbxcli"
+        ]
+    ],
+    "checkver": "github",
+    "autoupdate": {
```
[the remark this answers is not in the document]
Comment (Slach) [reply bucket/dbxcli.json:19]: fixed, could you check again?

**Review by HUMORCE — APPROVED** [review]

**Pull request merged**